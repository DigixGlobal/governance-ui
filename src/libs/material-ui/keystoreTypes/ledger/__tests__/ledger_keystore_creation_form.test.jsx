import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import LedgerKeystoreCreationForm from '../ledger_keystore_creation_form';

describe('<LedgerKeystoreCreationForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <LedgerKeystoreCreationForm
        formData={{}}
        formChange={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
