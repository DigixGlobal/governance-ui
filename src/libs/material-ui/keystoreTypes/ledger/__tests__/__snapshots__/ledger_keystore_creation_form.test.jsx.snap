// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LedgerKeystoreCreationForm /> renders correctly 1`] = `
<LedgerContianer
  renderReady={[Function]}
/>
`;
